src/components/core/Dropdown/Dropdown.tsx

```diff
@@ -117,10 +117,22 @@ const StyledDropdownBox = styled.div<StyledDropdownProps>`
 
   z-index: 1;
 
-  padding: 12px;
-  border: 1px solid ${colors.metal};
+  // min-width: 191px;
+
+  padding: 8px;
+  border: 1px solid ${colors.offBlack};
   width: max-content;
   background-color: ${colors.white};
+
+  button {
```

Comment:
would prefer to specify the style component that we're overriding where possible

for example if this button style override is for a TextButton component, we can do:
${TextButton} {

}

same for the text

Reply:
This was not working:

```
${TextButton} {

}
``` 

I think because `TextButton` is a component with props etc... it returned this error

<details>
<summary>error</summary>

```
Argument of type '({ className, text, onClick, disableTextTransform, disabled, dataTestId, }: Props) => Element' is not assignable to parameter of type 'Interpolation<ThemedStyledProps<Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "key" | keyof HTMLAttributes<HTMLDivElement>> & { ...; } & StyledDropdownProps, any>>'.
  Type '({ className, text, onClick, disableTextTransform, disabled, dataTestId, }: Props) => Element' is not assignable to type 'InterpolationFunction<ThemedStyledProps<Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "key" | keyof HTMLAttributes<HTMLDivElement>> & { ...; } & StyledDropdownProps, any>>'.
    Types of parameters '__0' and 'props' are incompatible.
      Property 'text' is missing in type 'Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "key" | keyof HTMLAttributes<HTMLDivElement>> & { ...; } & StyledDropdownProps & ThemeProps<...>' but required in type 'Props'.ts(2345)
TextButton.tsx(7, 3): 'text' is declared here.
```

</details>

So I exported the styled button component within `TextButton` and styled that instead ([changes](https://github.com/gallery-so/gallery/pull/870/commits/3f6ece0baae86dded84fbd4eff9f2bfb3df28041))